patterns/footer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Title: Default footer
+ * Slug: block-canvas/footer
+ * Categories: footer
+ * Block Types: core/template-part/footer
+ */
+?>
+
+<!-- wp:spacer {"height":"var(--wp--preset--spacing--80)"} -->
+<div style="height:var(--wp--preset--spacing--80)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group">
+    <!-- wp:group {"style":{"spacing":{"padding":{"top":"var(--wp--preset--spacing--60)","bottom":"var(--wp--preset--spacing--60)"}}}} -->
+    <div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)">
+        <!-- wp:paragraph {"align":"center"} -->
+        <p class="has-text-align-center">
+            <?php
+            /* Translators: WordPress link. */
+            $wordpress_link = '<a href="' . esc_url( __( 'https://wordpress.org', 'block-canvas' ) ) . '" rel="nofollow">WordPress</a>';
```

Comment:
The text domain should be "course" instead of "block-canvas", and on line 24 as well.

Reply:
Here also: https://github.com/Automattic/sensei-theme/pull/26/files#diff-1acd65465d61694f280f8f68d19e74df0e04128facf93e825c1ccdce9a89d1d0R22